fw-child/template/feedback.php

```diff
@@ -302,6 +302,124 @@ class="accordion-collapse collapse"
+								</div>
+								
+							</div>
+						
+							<div class="row row-cols-1 row-cols-lg-2 g-4">
+						
+								<div class="col d-flex flex-column">
+								
+									<label for="feedback" class="form-label"><?php _e ( 'Feedback', 'cdc' ); ?></label>
+									
+									<textarea name="feedback" rows="12" class="form-control flex-grow-1"></textarea>
+									
+								</div>
+								
+								<div class="col bg-light p-4 align-self-end">
+									
+									<div class="alert-container" style="display: none;">
+										<h5 class="alert-header"></h5>
+										<ul class="alert-messages"></ul>
+									</div>
+									
+									<div class="d-flex align-items-center mb-4">
+										<div class="captcha-img">
+											<?php
+											
+												$captcha_options = array (
+													'code_length' => 4,
+													'input_name' => 'support_captcha_code',
+													'input_id' => 'support_captcha',
+													'image_id' => 'support_captcha_img',
+													'icon_size' => 16,
+													'input_text' => __ ( 'Enter the characters shown in the image:', 'cdc' ),
+													'namespace' => 'support',
+												);
+												
+												// image
+												echo Securimage::getCaptchaHtml ( $captcha_options, Securimage::HTML_IMG );
+												
+												// refresh
+												echo Securimage::getCaptchaHtml ( $captcha_options, Securimage::HTML_ICON_REFRESH );
+												
+												// audio
+												echo Securimage::getCaptchaHtml ( $captcha_options, Securimage::HTML_AUDIO );
+													
+											?>
+										</div>
+										
+										<div class="">
+											<input type="hidden" name="namespace" value="<?php echo $captcha_options['namespace']; ?>">
+											
+											<label for="captcha_code" class="form-label small"><?php _e ( 'Enter the characters shown in the image', 'cdc' ); ?>:</label>
+											
+											<input type="text" name="captcha_code" id="captcha_code" class="form-control" placeholder="XXXX" size="4" maxlength="4" data-placement="bottom" title="<?php _e ( 'Non-valid entered characters. Please try again.', 'cdc' ); ?>">
+										</div>
+									</div>
+								
+									<button class="submit-btn btn btn-lg btn-primary rounded-pill d-block mx-auto" type="submit" id="feedback-data-submit"><?php _e ( 'Submit Feedback', 'cdc' ); ?> <i class="far fa-arrow-alt-circle-right ms-3"></i></button>	
+									
+								</div>
+								
+							</div>
+						</div>
+						
+					</div>
+					
+				</form>
+				
+			</div> 
+		</div> 
+	</div>
+
+	<div class="accordion-item"> 
+		<div id="accordion-head-beta-apps" class="accordion-header">
+			<div class="accordion-button collapsed row py-4 ps-0" 
+				data-bs-toggle="collapse"
+				data-bs-target="#beta-apps-support" 
+				aria-expanded="false" 
+				aria-controls="beta-apps-support"
+			>
+				<h3 class="col offset-2 mb-0">
+					<?php _e ( 'Beta Apps Support', 'cdc' ); ?>
+				</h3>
+			</div> 
+		</h2>
```

Comment:
This `</h2>` is the incorrect closing tag.

Note: there is the same problem in line 293.

Reply:
Good catch! I copied and pasted from 293 and didn't see that.